modules/nf-core/sourmash/gather/main.nf

```diff
@@ -0,0 +1,53 @@
+process SOURMASH_GATHER {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::sourmash=4.5.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/sourmash:4.5.0--hdfd78af_0':
+        'quay.io/biocontainers/sourmash:4.5.0--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(signature)
+    path(database)
```

Comment:
sourmash gather allow multiple databases to be passed per gather run. This is the preferred way to run e.g. all of the GenBank databases in one run. does encoding the database path this way allow multiple databases to be passed?

Reply:
`path` can indeed be a single file or multiple files. It might require different ways in the `script:` part to ensure correct command input.

Reply:
Okay, had to look up the gather command help, since one or more databases are just separate arguments, the way the process is written now it can already work for one or more databases. 🎉